src/Orchestration/Adapter/DockerAPI.php

```diff
@@ -177,6 +179,132 @@ protected function streamCall(string $url, int $timeout = -1): array
         ];
     }
 
+    /**
+     * Create Network
+     * 
+     * @param string $name
+     * @param bool $internal
+     * 
+     * @return bool
+     */
+    public function createNetwork(string $name, bool $internal = false): bool {
```

Comment:
```suggestion
    public function createNetwork(string $name, bool $internal = false): bool 
    {
```
PSR-12 guidelines recommend having the opening brace in a separate link 
refer : https://www.php-fig.org/psr/psr-12/
<img width="1158" alt="Screenshot 2021-12-12 at 1 38 02 PM" src="https://user-images.githubusercontent.com/20852629/145707416-920cbf51-6c7a-4c57-94a9-b093d05ca502.png">